Formula/o/odin.rb

```diff
@@ -81,6 +81,32 @@ def install
           "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
         cp "libraygui.dylib", buildpath/raylib_installpath
       end
+    else
+      raylib_installpath = "vendor/raylib/linux"
+
+      # Create symbolic links for GLFW and Raylib libraries
+      ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/"vendor/glfw/lib/darwin/libglfw3.a"
+      ln_s Formula["raylib"].lib/"libraylib.a", buildpath/raylib_installpath/"libraylib.a"
+
+      # Adjust the name of the Raylib dynamic library for Linux
+      ln_s Formula["raylib"].lib/"libraylib.so.4.5.0", buildpath/raylib_installpath/"libraylib.so.500"
+
+      resource("raygui").stage do
+        # Copy the raygui header and source files
+        cp "src/raygui.h", "src/raygui.c"
+
+        # Build the static library for raygui
+        system ENV.cc, "-c", "-o", "raygui.o", "src/raygui.c",
+               "-fpic", "-DRAYGUI_IMPLEMENTATION", "-I#{Formula["raylib"].include}"
+        system "ar", "-rcs", "libraygui.a", "raygui.o"
+        cp "libraygui.a", buildpath/raylib_installpath
+
+        # Build the shared library for raygui
+        system ENV.cc, "-o", "libraygui.so", "src/raygui.c",
+               "-shared", "-fpic", "-DRAYGUI_IMPLEMENTATION", "-lm", "-lpthread", "-ldl",
+               "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
+        cp "libraygui.so", buildpath/raylib_installpath
```

Comment:
```suggestion
        system ENV.cc, "-o", shared_library("libraygui"), "src/raygui.c",
               "-shared", "-fpic", "-DRAYGUI_IMPLEMENTATION", "-lm", "-lpthread", "-ldl",
               "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
        cp shared_library("libraygui"), buildpath/raylib_installpath
```
And make this unconditional. You can make the exact flags conditional on OS.